war/src/main/java/io/lumeer/engine/controller/DocumentFacade.java

```diff
@@ -81,13 +81,15 @@
     *
     * @param collectionName
     *       the name of the collection where the document will be created
+    * @param collectionCode
+    *       collection code
     * @param document
     *       the DataDocument object representing a document to be created
     * @return the id of the newly created document
     * @throws DbException
     *       When there is an error working with the database.
     */
-   public String createDocument(final String collectionName, final DataDocument document) throws DbException {
+   public String createDocument(final String collectionName, final String collectionCode, final DataDocument document) throws DbException {
```

Comment:
Why do we need `collectionName` parameter here? It does not uniquely identify collections.